src/libraries/Common/src/System/Security/Cryptography/SlhDsa.cs

```diff
@@ -677,25 +816,55 @@ public int ExportSlhDsaSecretKey(Span<byte> destination)
             return secretKeySizeInBytes;
         }
 
+        /// <summary>
+        ///   Exports the current key in the FIPS 205 secret key format.
+        /// </summary>
+        /// <returns>
+        ///   The secret key.
+        /// </returns>
+        /// <exception cref="CryptographicException">
+        ///   <para>The current instance cannot export a secret key.</para>
+        ///   <para>-or-</para>
+        ///   <para>An error occurred while exporting the key.</para>
+        /// </exception>
+        /// <exception cref="ObjectDisposedException">The object has already been disposed.</exception>
+        public byte[] ExportSlhDsaSecretKey()
+        {
+            ThrowIfDisposed();
+
+            byte[] destination = new byte[Algorithm.SecretKeySizeInBytes];
+            ExportSlhDsaSecretKeyCore(destination);
+            return destination;
+        }
+
         /// <summary>
         ///   Generates a new SLH-DSA key for the specified algorithm.
         /// </summary>
         /// <returns>
         ///   The generated object.
         /// </returns>
+        /// <exception cref="ArgumentNullException">
+        ///   <paramref name="algorithm" /> is <see langword="null" />.
+        /// </exception>
+        /// <exception cref="CryptographicException">
+        ///   An error occurred generating the SLH-DSA key.
+        /// </exception>
+        /// <exception cref="PlatformNotSupportedException">
+        ///   The platform does not support SLH-DSA. Callers can use the <see cref="IsSupported" /> property
+        ///   to determine if the platform supports SLH-DSA.
+        /// </exception>
         public static SlhDsa GenerateKey(SlhDsaAlgorithm algorithm)
         {
             ArgumentNullException.ThrowIfNull(algorithm);
             ThrowIfNotSupported();
-
```

Comment:
Don't like my blank line? :)

I'm a big believer in "code paragraphs".  This was styled as "precondition exceptions", blank line to indicate that's over, then "one line implementation".

Is it necessary? No. Is it anything that anyone can articulate in a style guide? Not really.  Is it a thing about which reasonable people can argue? Definitely.

Certainly if it was the only change I'd say to put it back.  I don't super care, but felt like explaining why it was there, and am slightly annoyed by its existence as an "unrelated change"